keps/976-plain-pods/README.md

```diff
@@ -494,22 +461,32 @@ spec:
       spec:
         containers:
         - name: job
-          image: hello-world
+          image: driver
           resources:
             requests:
               cpu: 1m
-  - count: 10
+  - count: 2
     name: worker
     template:
       spec:
         containers:
         - name: job
-          image: hello-world
+          image: worker
           resources:
             requests:
               cpu: 1m
+              vendor.com/gpu: 1
 ```
 
+**Caveats:**
+
+If the number of different sha256s obtained from the groups of Pods, Workload creation will fail.
+This generally shouldn't be a problem, unless multiple Pods (that should be considered the same
+from an admission perspective) have different label values or reference different volume claims.
```

Comment:
Kueue recognizes multiple pods as the same shape even though pods have different commands and args. 
Because Kueue doesn't consider commands and args when grouping multiple pods as the same shape.
Is it right?

Reply:
> But users should be encouraged to use Job or JobSet if they face this specific problem.

Yes, it definitely is right.